lua/caramba/git.lua

```
@@ -519,30 +524,181 @@ M.review_code = function()
+8. Test coverage adequacy (if test files found)
+9. Documentation completeness
+
+Consider the semantic context and how this code interacts with:
+- Imported modules and their APIs
+- Related files in the project (especially tests)
+- Available symbols and their usage
+- Overall project architecture
+- Recent changes and git history
+
+Provide specific line references and actionable suggestions. If tests exist, comment on test coverage gaps.
+]], language, language, review_type, table.concat(context_sections, "\n"), semantic_context, related_files, 
```

Luacheck (1.2.0)

[warning] line contains trailing whitespace

(W612)